docs/root/configuration/http/http_filters/grpc_http1_reverse_bridge_filter.rst

```diff
@@ -104,10 +105,15 @@ How to disable HTTP/1.1 reverse bridge filter per route
       type: LOGICAL_DNS
       dns_lookup_family: V4_ONLY
       lb_policy: ROUND_ROBIN
-      hosts:
-        - socket_address:
-            address: localhost
-            port_value: 4630
+      load_assignment:
```

Comment:
For the new fragments that you wrote, did you verify they worked in some kind of test harness, e.g. that this load assignment would parse? It looks reasonable, but, would be reassuring to have some kind of machine check.

Long term we want https://github.com/envoyproxy/envoy/issues/8837, but that's outside the scope of this PR; I'd be reassured if at least they were known to parse as valid at least once.

Reply:
I did not, agree it would be a good idea.

Reply:
Ping on this one. I think it would be good to at least verify one example of each kind of change you've made to ensure there isn't typo/subtle failure. What you could do is take https://github.com/envoyproxy/envoy/blob/master/configs/google_com_proxy.v2.yaml and copy+paste in fragments with some surrounding context, verify it can be loaded with `envoy -c`. I don't think we need anything beyond that.

Is this a tractable amount of work or is it going to be too crazy?